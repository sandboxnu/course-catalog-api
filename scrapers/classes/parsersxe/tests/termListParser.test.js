import TermListParser from "../termListParser";

describe("termListParser", () => {
  it("pulls out relevant data", () => {
    const list = [
      {
        code: "202034",
        description: "Spring 2020 CPS Semester",
      },
      {
        code: "202032",
        description: "Spring 2020 Law Semester",
      },
      {
        code: "202030",
        description: "Spring 2020 Semester",
      },
    ];
    expect(TermListParser.serializeTermsList(list)).toMatchSnapshot();
  });
});
