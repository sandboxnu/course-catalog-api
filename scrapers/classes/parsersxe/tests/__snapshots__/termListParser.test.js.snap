// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`termListParser pulls out relevant data 1`] = `
[
  {
    "active": true,
    "host": "neu.edu",
    "subCollege": "CPS",
    "termId": "202034",
    "text": "Spring 2020 Semester",
  },
  {
    "active": true,
    "host": "neu.edu",
    "subCollege": "LAW",
    "termId": "202032",
    "text": "Spring 2020 Semester",
  },
  {
    "active": true,
    "host": "neu.edu",
    "subCollege": "NEU",
    "termId": "202030",
    "text": "Spring 2020 Semester",
  },
]
`;
